### ISSUE TITLE
Increase mouse sensitivity even more

### ISSUE THREAD

**Issue by kooskaspers** [hyprwm/Hyprland#2056]
I'd like to increase the sensitivity of my mouse, which is a Logitech trackball (mx-ergo). I've already set the sensitivity to the maximum value of 1.0:
```
device:mx-ergo {
    scroll_method=on_button_down
    natural_scroll=1
    sensitivity=1.0
    accel_profile=adaptive
    scroll_method=on_button_down
    scroll_button=273
}
```

but unfortunately, this is still way to slow to be able to move the mouse across the screen. I'd like to request a feature which makes it possible to increase the speed even more. With sway, this configuration was enough to make it usable:
```
input "1133:45085:MX_Ergo" {
     scroll_button button3
     scroll_method on_button_down
     middle_emulation true
     natural_scroll true
     pointer_accel 0.8
     scroll_factor 0.7
     accel_profile adaptive
   }
```

**Comment by kooskaspers** [comment]
Would it be possible to add another layer between the input device and the compositor to increase the mouse speed?

**Comment by vaxerski** [comment]
wouldn't make sense, as you'd start skipping pixels. Increase the dpi of your mouse.

**Comment by kooskaspers** [comment]
Sway seems to handle the mouse input differently, where it does make the mouse move way more sensitive. Is there any way we can make the input of the mouse more sensitive without having to change the dpi of the mouse itself? The DPI of my mouse isn't configurable :(.

**Comment by vaxerski** [comment]
sway seems to do the same, and read the inputs the same way, unless you set `input:force_no_accel` to 1 in hl.

**Comment by kooskaspers** [comment]
I seem to get that. I haven't set force_no_accel to 1, so it should read my settings. If sway would read the inputs the same way, I should be able to copy my configuration from sway and have the same experience with hyperland. The two variables which influence the mouse speed should be:
```
sway: accel_profile adaptive
sway: pointer_accel 0.8
```
translating these to hyprland would be:
```
hyprland: accel_profile=adaptive
hyprland: sensitivity=0.8
```

But that's gives unfortunately still a total different experience in mouse sensitivity. Am I overlooking something?

**Comment by vaxerski** [comment]
> Would it by any chance possible to support the 'custom' acceleration profile? [link](https://wayland.freedesktop.org/libinput/doc/latest/pointer-acceleration.html#ptraccel-profile-custom)
This profile accepts 2 variables, comma seperated. See link.

Yes

**Comment by vaxerski** [comment]
done in 0859944c9ab7855a68ab9126d0f527f857d9d40d

although doubt custom will help you as it still cant go over 1

**Comment by kooskaspers** [comment]
Great, thank you @vaxerski  for testing this out with me.
Just compiled the latest git version and trying out the custom profile. I'm not 100% sure how to pass the 2 variables, since I don't see a new variable being introduced regarding those 2 floats. Is it being read from the accel_profile string?, like:

Can the custom accel_profile be set like:
```accel_profile=custom 0 1
```

**Comment by vaxerski** [comment]
![image](https://user-images.githubusercontent.com/43317083/236559203-bedcefe7-af67-48ec-9187-e9b09ef7145e.png)

rtfw https://wiki.hyprland.org/Configuring/Variables/#decoration

**Comment by kooskaspers** [comment]
@vaxerski, I tried multiple settings:

```
custom -1 0.0 1.0
#sensitivity=1.0

custom -1 1.0 0.0
#sensitivity=1.0

custom 1 0.0 1.0
#sensitivity=1.0

custom 1 1.0 0.0
#sensitivity=1.0

custom -1 0.0 1.0
sensitivity=1.0

custom -1 1.0 0.0
sensitivity=1.0

custom 1 0.0 1.0
sensitivity=1.0

custom 1 1.0 0.0
sensitivity=1.0
```

but I don't feel any difference at all by making use of the newly introduced custom accel_profile.

Output of devices:

```
mice:
	Mouse at 3dd73240:
		bcm5974
			default speed: 0.000000
	Mouse at 3dd69d70:
		keyd-virtual-pointer
			default speed: 0.000000
	Mouse at 3ff230c0:
		logitech-mx-ergo-multi-device-trackball-
			default speed: 0.000000

Keyboards:
(skipped)

Tablets:

Touch:

Switches:
	Switch Device at 3daa1240:
		Lid Switch

```

config:

```
device:logitech-mx-ergo-multi-device-trackball- {
    #scroll_button=button3
    scroll_method=on_button_down
    #middle_emulation=true
    #pointer_accel=0.8
    #scroll_factor=0.7
    natural_scroll=1
    #sensitivity=1.0
    accel_profile=custom 1 0.0 1.0
    scroll_method=on_button_down
    scroll_button=273
}
```

I'm on hyprland-git commit r2580.71ef1bde-3
Anything I'm missing?

**Comment by vaxerski** [comment]
not really, the sensitivity difference makes no sense to me, furthermore it hsould be handled by libinput

**Comment by kooskaspers** [comment]
I've changed laptops, and I'm facing the same issue nu with a touchpad. It's way to sloo on these high resolutions. Config:

```
device:bcm5974 {
 tap-to-click=1
 natural_scroll=1
 sensitivity=1.0
 accel_profile=adaptive
}
```

This is the maximum speed I'm able to configure. Anything left which I can try to make my mouse and touchpad more responsive?

**Comment by kooskaspers** [comment]
alright. my fault. "sensitivity=0.25" was placed in the General section :|

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
